internal/mode/static/nginx/conf/nginx.conf

```diff
@@ -4,6 +4,7 @@ events {}
 
 pid /var/run/nginx/nginx.pid;
 error_log stderr debug;
+worker_processes auto;
```

Comment:
I set this so I could easily test with more than 1 worker process - but should we set this as the default?

Reply:
I'm setting that value as part of #557